zzzz_modular_occulus/code/modules/projectiles/guns/projectile/rifle/whitenail.dm

```diff
@@ -33,7 +34,7 @@
 	START_PROCESSING(SSobj, src)
 	update_icon()
 
-/obj/item/weapon/gun/projectile/automatic/whitenail/Process() // increments counter by one per tick until it reaches 
+/obj/item/weapon/gun/projectile/automatic/whitenail/Process() // increments counter by one per tick until it reaches gun's rebuild_time var
```

Comment:
I know it's rather late to be commenting this since the White Nail is already in the code, but we may want to take another look at how it handles ammo generation and processing.

A slightly more effective method in this case would be to only have the gun process when it is not full on ammo - stopping when it gets full, and starting again after being fired.